pkg/services/container/announcement/load/storage/storage.go

```diff
@@ -118,6 +125,22 @@ func (s *Storage) Iterate(f loadcontroller.UsedSpaceFilter, h loadcontroller.Use
 	return
 }
 
+// NewEpoch notifies [Storage] about epoch counter updating.
+// Used to remove unused estimations. See [Prm.EstimationsLifeCycle].
+// Blocking operation.
+func (s *Storage) NewEpoch(e uint64) {
```

Comment:
i'd add verb to the name, e.g. `SubmitNewEpoch`

Reply:
Or `EpochEvent`.